provider/storage_benchmark.py

```diff
@@ -213,6 +213,7 @@ def _find_exe_file():
         if not utils_misc.wait_for(
                 lambda: _find_exe_file(), timeout, step=3.0):
             raise TestError('Failed to install fio under %.2f.' % timeout)
+        self.session.cmd_output(r'cd ..', timeout=timeout)
```

Comment:
IMO, update the L204 to remove `CD` and use `DIR` to check the `dst` is possible.

Reply:
With your latest update, this line can be removed now.

Reply:
oh，Yes!  
Updated, thanks